src/components/labwareResult/LabwareResult.tsx

```diff
@@ -56,64 +58,109 @@ export default function LabwareResult({
   );
 
   const { labwareResult } = current.context;
+
   const sampleResults = mapify(labwareResult.sampleResults, "address");
+  const slotMeasurements = labwareResult.slotMeasurements
+    ? mapify(labwareResult.slotMeasurements, "address")
+    : new Map<string, SlotMeasurementRequest>();
 
   useEffect(() => {
     onChange(labwareResult);
   }, [labwareResult, onChange]);
 
+  const isMeasurementExist = "slotMeasurements" in labwareResult;
+
+  /**Ensure Tissue Coverage value is in the range 0 to 100 inclusive**/
+  const validateMeasurementField = (value: string) => {
+    const regEx = /^(^$|[0-9]|([1-9][0-9])|100)$/;
```

Comment:
You're about to try and convert the value to a number, so accepting the empty string `^$` does not make sense here. `/^([0-9]|([1-9][0-9])|100)$/` would be OK, but you could simply use `/^[0-9]+$/` to check it's made up of one or more digits.

Reply:
Thought this would avoid entering values like 05 etc  to give visual effect. But as you said functionality wise that won't be a problem since we are converting to an integer. 
I have put empty string option because acceptance test says it can be an empty field. So if the user enters a value and then deletes it or use backspace it will allow to remove the text completely. 
But again, for this case, we can always check the string length to 0 and return a true without using regex.

Reply:
@khelwood Updated regex and pushed in changes.